packages/container/print.js

```diff
@@ -36,8 +40,25 @@ function waitForEnter() {
   })
 }
 
+function formatLine(content, color) {
+  let result = colorCode(color)
+  let codeBlock = false
+  for (const c of content) {
+    switch (c) {
+      case '`':
+        result += colorCode(codeBlock ? color : 'green')
```

Comment:
```suggestion
        result += colorCode(codeBlock ? color : 'yellow')
```

Reply:
i think that's fine

Reply:
You mean we don't need to note code in hint content?